### ISSUE TITLE
The transcript runner hasn't produced `ucm` feedback for this `unison` block

### ISSUE THREAD

**Issue by mitchellwrosen** [unisonweb/unison#5407]
I ran into this bug while writing a transcript recently. My workaround for now is to repeat the same `unison` block twice. You can see that feedback from `ucm` is empty the first time (incorrect), and non-empty the second time (correct).

https://github.com/unisonweb/unison/pull/5404/files#diff-0b3f70e4ed01aa42136cec276d6414f714b2b1caa1a8a601f6944b5b0f0c9674R77-R108

**Comment by aryairani** [comment]
Looks like a duplicate of #4598

**Comment by mitchellwrosen** [comment]
Yeah, it does... I'm inclined to leave this ticket open until that's fixed, though, just in case they are different bugs (unlikely)

**Comment by aryairani** [comment]
<img width="1054" alt="image" src="https://github.com/user-attachments/assets/7d082ebb-8257-482a-ae0d-0d47625b4201">
Seems to be literally the same: `edit` is used in a transcript and then the next `unison` block is ignored.
